packages/fxa-auth-server/lib/senders/emails/global.scss

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Fonts
+$font-sans: sans-serif;
+
+// Colors
+$blue-400: #0090ed;
+$white: #fff;
+$grey-400: #6d6d6e;
+
+// Spacing values for margin and padding
+$s-0: 0px;
+$s-3: 12px;
+$s-5: 20px;
+$s-6: 24px;
+$s-10: 40px;
+
+// Font-size and line-height
+.text-xs {
+  font-size: 12px !important;
+  line-height: 18px !important;
+}
+.text-sm {
+  font-size: 14px !important;
+  line-height: 20px !important;
+}
+.text-lg {
+  font-size: 18px !important;
+  line-height: 28px !important;
+}
+.text-xl {
+  font-size: 20px !important;
+  line-height: 28px !important;
+}
+
+// Utility classes
+.font-sans {
+  font-family: $font-sans !important;
+}
+
+.text-blue-400 {
+  color: $blue-400;
+}
+
+.mt-6 {
+  margin-top: $s-6 !important;
+}
+
+.mb {
+  &-3 div {
```

Comment:
I think we need to explicitly apply mirgin to the div of the class so I had to do it this way

Reply:
Ah interesting. If/when we need `mb-3` _without_ `div`, we should update this line to be `&-3, &-3 div`.